src/services/hasura/hasura.service.ts

```
@@ -281,19 +281,26 @@ export class HasuraService {
-      }
-    }
+  async insertCacheData(arrayOfObjects: any[], bpps: string[]) {
+	this.logger.log(
+		`Starting cache refresh for ${bpps.length} BPP(s): ${bpps.join(', ')}`,
+	);
+	this.logger.log(
+		`Deleting existing records and inserting ${arrayOfObjects.length} new records`,
+	);
+
+	// First delete existing records for the specified BPPs only
+	if (arrayOfObjects.length > 0 ) {
```

ESLint

[error] Expected indentation of 2 tabs but found 1.

(indent)